src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -45,11 +49,13 @@
      * @param domain       tenant's domain.
      * @param clientId     the application's client id.
      * @param clientSecret the application's client secret.
+     * @param options      configuration options for this client instance.
```

Comment:
The method JavaDoc itself could use updating, as it directly references the domain, clientId, and secret (but not the configuration options). Also would be good to add a `@see` reference to the existing constructor that does not accept a `ClientOptions` instance.

Reply:
That's a good point

Reply:
I added a simple `@see #method(with, params, here)`. It's clickable at least on the IDE and links to the right constructor.